langchain-llamaindex-slackbot/src/main.py

```diff
@@ -56,11 +55,19 @@
 app = App(token=SLACK_BOT_TOKEN)
 
 # Langchain implementation
-template = """
-    Using only the following context answer the question at the end. If you can't find the answer in the context below, just say that you don't know. Do not make up an answer.
+template = (
+    """
+    Using only the following context answer the question at the end. If you
+    can't find the answer in the context below, just say that you don't know. Do
+    not make up an answer. The latest version of ZenML that you have
+    been trained on is 
+    """
+    + LATEST_ZENML_VERSION
+    + """
```

Comment:
It actually doesn't affect the answers, unfortunately. Even adding this and not touching the data still had it say that latest version was 0.5.1... But it can't hurt so I left it in.